examples/hello_world/client/nodejs/index.js

```diff
@@ -1,24 +1,54 @@
 const fs = require('fs');
-const protoLoader = require('@grpc/proto-loader');
+const protobufjs = require('protobufjs');
 const grpc = require('@grpc/grpc-js');
+const grpcProtoLoader = require('@grpc/proto-loader');
 
 const CERT_PATH = __dirname + '/../../../certs/local/ca.pem';
```

Comment:
If I will run this example from the `oak/` directory, what will be in the `__dirname` ? Is it always a directory of this file?

Reply:
yes :). Relative to this file. 

I prefer paths relative to the project root, but they'd require adding a transpiler